examples/raw.rs

```diff
@@ -0,0 +1,64 @@
+extern crate futures;
+extern crate tikv_client;
+
+use futures::future::Future;
+use tikv_client::raw::Client;
+use tikv_client::*;
+
+fn main() {
+    let config = Config::new(vec!["127.0.0.1:3379"]);
+    let raw = raw::RawClient::new(&config)
+        .wait()
+        .expect("Could not connect to tikv");
+
+    let key: Key = b"Company".to_vec().into();
+    let value: Value = b"PingCAP".to_vec().into();
+
+    raw.put((Clone::clone(&key), Clone::clone(&value)))
```

Comment:
You can use `key.clone()` here to have it read less repetitively.

Reply:
Oops. I remember Arc::clone is preferred over calling clone directly and thought it applies to Clone::clone. Looks like it is not the case for Clone. Thanks for pointing out.